project/src/main/webapp/maps.js

```diff
@@ -0,0 +1,42 @@
+/* Utilize JavaScript Maps API and Places API to display an embedded map through a info window.*/
+function initMap() {
+    if (classifyQuestionType(document.getElementById('query').value) === 'Place'){
+        document.getElementById("map").style.display = "block";
+        let infowindow = new google.maps.InfoWindow();
+        let map = new google.maps.Map(document.getElementById("map"), {
+            center: google.maps.LatLng(-33.867, 151.195),
+            zoom: 15
+        });
+
+        var search = document.getElementById('displayAnswer').value;
+        const request = {
+            query: search,
+            fields: ["name", "geometry"]
+        };
+
+        let service = new google.maps.places.PlacesService(map);
+        service.findPlaceFromQuery(request, (results, status) => {
+            if (status === google.maps.places.PlacesServiceStatus.OK) {
+            for (let i = 0; i < results.length; i++) {
```

Comment:
Fix indentation. You can format the entire file in the cloud console editor with right click > Format Document